app/controllers/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+class UsersController < ApplicationController
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    user_params = params.require(:user).permit(:first_name, :last_name, :email,
+                                            :password, :password_confirmation)
```

Comment:
eventhough you are calling this one time, still better to put it in a private user_params method
 Add a line note